rabbitmq-queue-dashboard: fixed an issue with the usage of the datasource, added meaningful Legends and refresh to 10s

sherifkayad opened the pull request:
…urce, added meaningful Legends and refresh to 10s

This closes #

**Note to reviewers:** remember to look at the commits in this PR and consider if they can be squashed

## Summary Of Changes
**rabbitmq-queue-dashboard:**
- fixed an issue with the usage of the datasource
- added meaningful Legends
- added refresh to 10s

## Additional Context

## Local Testing

Please ensure you run the unit, integration and system tests before approving the PR.

To run the unit and integration tests:

```
$ make unit-tests integration-tests
```

You will need to target a k8s cluster and have the operator deployed for running the system tests.

For example, for a Kubernetes context named `dev-bunny`:
```
$ kubectx dev-bunny
$ make destroy deploy-dev
# wait for operator to be deployed
$ make system-tests
```

DanielePalaia commented:
Hi @sherifkayad Thank you for the PR. Just to help us on the review, could you explain what issue you are experimenting with this dashboard and what improvements are you bring in? (Maybe also opening an issue and link it to this PR would be nice, and why not posting the grafana images if possibile).

sherifkayad commented:
@DanielePalaia The issue I had was if one would have a Prometheus DS not with the name `Prometheus` then this Dashboard would break; so I fixed that.

In addition, I added some legends to the Graph such that one could understand which value (bar) is the messages and which is the consumers.

My bad I didn't add some screenshots .. There you go.

![Screenshot 2023-03-30 170525](https://user-images.githubusercontent.com/8362784/228881031-e89ab04f-bed6-48fd-b6c7-086e8166921c.png)

Zerpet commented:
Tried out the changes in this PR. This is the before:

![Screenshot 2023-04-27 at 17 07 47](https://user-images.githubusercontent.com/1515757/234935148-31ec433e-6f1d-449e-b066-b1f9778491e6.png)

And after:

![Screenshot 2023-04-27 at 17 17 31](https://user-images.githubusercontent.com/1515757/234935221-0e8e7c85-2908-446b-8d67-590181f81288.png)

Zerpet reviewed (approved):
Thank you!

sherifkayad commented:
@Zerpet thanks for the feedback .. glad to be able to contribute something to this awesome repo!
